modules/ROOT/pages/import-api-specification-local-file.adoc

```diff
@@ -0,0 +1,37 @@
+= Importing an API Specification from a Local File
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Anypoint Studio allows you to import an API specification from your local machine and import it either into a new Mule Project, or an existing one.
+
+This task only works for RAML types of API specifications.
```

Comment:
DITTO edits to same text earlier in topic set:

I suggest moving this point into the intro, by specifying RAML there (if we mean the same thing by "specification" as by "definition":

"Anypoint Studio enables you to import a RAML API definition from Design Center into either a new Mule project or an existing one."

Perhaps we could add the size limitation here as well:

"Anypoint Studio enables you to import a RAML API definition of fewer than 4,000 lines from Design Center into either a new Mule project or an existing one."

Just ideas....